### PR TITLE
add your-HenokB

### LINKED ISSUE #371: Henok B Ademtew
henok

### PR DESCRIPTION
**Pull request by HenokB**
resolve #371

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -599,4 +599,10 @@
     "imageUrl": "https://user-images.githubusercontent.com/29686102/106919215-f0711a00-672f-11eb-88f6-8ab44f67b911.png",
     "issueNumber": 313
   }
```
Comment (schmelto) [inline data.json:601]: You should at a `, ` here I think

**Review by schmelto — CHANGES_REQUESTED** [review]

File: data.json (hunk 1 above)
Comment (eddiejaoude) [reply data.json:601]: Thank you for reviewing @schmelto 👍

File: data.json [hunk 2]
```diff
@@ -599,4 +599,10 @@
...
+  {
+    "name": "Henok B Ademtew",
+    "githubUsername": "HenokB",
+    "imageUrl": "https://user-images.githubusercontent.com/46082799/107974254-4f326100-6f84-11eb-8cbb-105cd0d59189.png",
+    "issueNumber": 351
```
Comment (AllanRegush) [inline data.json:606]: What is your Issue number? #351 is a PR

**Review by AllanRegush — COMMENTED** [review]
Looking Good 👍

**Comment by AllanRegush** [comment]
@HenokB Just Checking in. Do you need any help? If you do let us know 🤓

File: data.json (hunk 2 above)
Comment (Vyvy-vi) [reply data.json:606]: I think their issue is #371

**Review by praveenscience — CHANGES_REQUESTED** [review]
Correct your issue number and please fix the error on the code.
image

**Comment by AllanRegush** [comment]
Hi @HenokB Just Checking in. If you need any help let use know :nerd_face:

File: data.json (hunk 1 above)
Comment (schmelto) [reply data.json:601]:
```suggestion
  },
```

File: data.json (hunk 2 above)
Comment (schmelto) [reply data.json:606]:
```suggestion
    "issueNumber": 371
```

**Review by schmelto — CHANGES_REQUESTED** [review]

**Review by schmelto — APPROVED** [review]

File: data.json [hunk 3]
```diff
@@ -611,6 +611,12 @@
     "imageUrl": "https://user-images.githubusercontent.com/29686102/106919215-f0711a00-672f-11eb-88f6-8ab44f67b911.png",
     "issueNumber": 313
   },
+  {
+    "name": "Henok B Ademtew",
```
Comment (Panquesito7) [inline data.json:615]: I copied their name from their GitHub bio, and I think it'd be better.

```suggestion
    "name": "Henok Biadglign Ademtew",
```

Or

```suggestion
    "name": "Henok B. Ademtew",
```

too.

**Pull request merged**